airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/schemas/ticket_metrics.json

```diff
@@ -131,7 +131,8 @@
       "type": ["null", "string"]
     },
     "initially_assigned_at": {
-      "type": ["null", "string"]
+      "type": ["null", "string"],
+      "format": "datetime"
```

Comment:
does this need to be `date-time`?

Reply:
/publish connector=source-zendesk-support

Reply:
fixed